ignite/internal/plugin/consumer.go

```diff
@@ -0,0 +1,39 @@
+package plugininternal
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/ignite/cli/v28/ignite/pkg/errors"
+	"github.com/ignite/cli/v28/ignite/services/plugin"
+)
+
+// TODO use released version of app-consumer
+const consumerPlugin = "github.com/ignite/apps/consumer"
+
+// ConsumerWriteGenesis writes validators in the consumer module genesis.
+// NOTE(tb): Using a plugin for this task avoids having the interchain-security
+// dependency in Ignite.
+func ConsumerWriteGenesis(ctx context.Context, c plugin.Chainer) error {
+	_, err := Execute(ctx, consumerPlugin, []string{"writeGenesis"}, plugin.WithChain(c))
```

Comment:
We have a PKG here to deal with big genesis files and also write things to Genesis but was removed

`ignite/pkg/cosmosutil/genesis/genesis.go`

the network still use: https://github.com/ignite/cli-plugin-network/blob/eaabb34da81877bf7b33b5143c35d1ddb489ee48/network/networkchain/prepare.go#L121-L121

Reply:
Yes, but it was removed by mistake. Because the network app still uses it. I will rollback it when I have time to work in the network app